components/MultiChoiceForm.tsx

```diff
@@ -0,0 +1,160 @@
+import { AddAlt } from "@carbon/icons-react";
+import { LinkButton } from "@components/Button";
+import { StyledLabel } from "@components/inputs/styles";
+import { XCircleIcon } from "@heroicons/react/solid";
+import useGovernanceAssets from "@hooks/useGovernanceAssets";
+import useRealm from "@hooks/useRealm";
+import Input from '@components/inputs/Input';
+import { AssetAccount } from "@utils/uiTypes/assets";
+import { useState } from "react";
+import GovernedAccountSelect from "../pages/dao/[symbol]/proposal/components/GovernedAccountSelect";
+
+const MultiChoiceForm = ({
+    multiChoiceForm,
+    updateMultiChoiceForm,
+    isMultiFormValidated,
+    multiFormErrors,
+    updateMultiFormErrors
+} : {
+    multiChoiceForm: {
+        governedAccount: AssetAccount | undefined
+        options: string[]
+    }
+    updateMultiChoiceForm: any
+    isMultiFormValidated: boolean
+    multiFormErrors: any
+    updateMultiFormErrors: any
+}) => {
+    const {ownVoterWeight} = useRealm()
+    const { assetAccounts } = useGovernanceAssets()
+
+    const [hideNotaButton, setHideNotaButton] = useState(false)
```

Comment:
this is redundant state and shouldn't exist. you can just do something like

`const hideNotaButton = options.includes("None of the Above")`

Reply:
nvm, see discord comments